lib/web_ui/dev/test_platform.dart

```diff
@@ -565,15 +547,27 @@ class BrowserPlatform extends PlatformPlugin {
     canvasKitVariant: "${getCanvasKitVariant()}",
   };
 </script>
-      ''' : '<script src="/flutter_js/flutter.js"></script>';
+<script src="/test_dart2wasm.js"></script>
+      ''' : '''
+<script src="/flutter_js/flutter.js"></script>
+<script type="module">
+  import { runTest } from "/test_dart2js.js";
+
+  runTest({
+    canvasKitBaseUrl: "/canvaskit/",
+    // Some of our tests rely on color emoji
+    useColorEmoji: true,
+    canvasKitVariant: "${getCanvasKitVariant()}",
+  });
```

Comment:
![I've always dreamed of seeing the future](https://media4.giphy.com/media/ZhESFK96NxbuO1yDgy/giphy.gif?cid=ecf05e47egkfaq2eljdv3s6jrx20z6ys7klgot2ea1pmkqox&ep=v1_gifs_search&rid=giphy.gif&ct=g)

(The future of bootstrapping with flutter.js right here 😋)